go-app/main.go

```diff
@@ -99,23 +106,37 @@ func homeHandler(w http.ResponseWriter, r *http.Request) {
 	case "UploadConfig":
 		configs, err := settings.UploadConfigFile(r, project_id, cloudConfigs)
 		if err != nil {
+			feedBack.Error = true
+			feedBack.Content = err.Error()
+			feedbackTempl.Execute(w, feedBack)
```

Comment:
These three lines are duplicated a lot. I'd pull this into a private helper method that takes a boolean and a string as parameters, setting error and content respectively. Then calling feedbackTempl.Execute(w, feedback)